src/index.js

```diff
@@ -1,41 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import List, { Hours } from './Compo';
-import App from './App';
+import CustomersList from './CustomersList';
```

Comment:
You wrote `CustomersList` (customer`s` plural and their list, double times telling bunch of customers which is redundant). Just CustomerList (with no plural) is enough.